src/analysis/retail/monk/mistweaver/CONFIG.tsx

```diff
@@ -9,7 +9,7 @@ const config: Config = {
   contributors: [],
   expansion: Expansion.Dragonflight,
   // The WoW client patch this spec was last updated.
-  patchCompatibility: null,
+  patchCompatibility: '10.0.2',
   isPartial: false,
```

Comment:
If DF isn't fully supported yet, either set `isPartial` to true, or leave patchCompatability as 9.2.7 (analysis will run but will display 'not updated for latest patch' message)

Reply:
Set isPartial to true